try:
    print(8/2)
except:
    print("exception raised")